package.json

```diff
@@ -491,6 +511,21 @@
         "description": "Restarts a composition of containers",
         "category": "Docker"
       },
+      {
+        "command": "vscode-docker.create-ACR-Registry",
+        "title": "Create Registry",
```

Comment:
"title": "Create Azure Registry..."

The "Azure" part is important because from the command palette the context would otherwise be unclear.  The "..." lets the user know we will be asking for more information.

Reply:
Yes, our pm suggested that we could just leave it so the commands would show up as ACR : Create Registry, ACR : Delete Repository, etc …

Reply:
@fiveisprime Matt, what are your thoughts on having the ACR commands use a different category than the rest?  Personally I'd rather they all start with "Docker: ".

(Esteban - I misspoke earlier about the category matching the node - I was referring to the command title, not the category).

I.e., I'd suggest:
   Docker: Create ACR Registry
   Docker: Delete ACR Repository
...

Reply:
I would rather than start with "Docker:" as well; I like the suggestion of "Docker: Create ACR Registry" and so on. Is that clear enough? 

Should we still abbreviate in this case? For example, would it make more sense to say "Docker: Create Azure Container Registry"?

Reply:
I think either would be fine, I'll try to message my team's PM to see what his opinion is.